src/System.Private.CoreLib/shared/System/Collections/Generic/Comparer.cs

```diff
@@ -10,21 +10,18 @@
 namespace System.Collections.Generic
 {
     [Serializable]
-    [TypeDependencyAttribute("System.Collections.Generic.ObjectComparer`1")]
     [System.Runtime.CompilerServices.TypeForwardedFrom("mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")] 
-    public abstract class Comparer<T> : IComparer, IComparer<T>
+    public abstract partial class Comparer<T> : IComparer, IComparer<T>
     {
-        // To minimize generic instantiation overhead of creating the comparer per type, we keep the generic portion of the code as small
-        // as possible and define most of the creation logic in a non-generic class.
-        public static Comparer<T> Default { get; } = (Comparer<T>)ComparerHelpers.CreateDefaultComparer(typeof(T));
-
+#if !PROJECTN
```

Comment:
Why this ifdef? `Comparer<T> Create(Comparison<T> comparison)` looks exactly same in ProjectN/CorerRT.

Reply:
Good find, just leftover from files diffing